modules/rosa-sdpolicy-am-aws-compute-types-graviton.adoc

```diff
@@ -0,0 +1,186 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types.adoc
+:_mod-docs-content-type: CONCEPT
+[id="rosa-sdpolicy-aws-instance-types-graviton_{context}"]
+= AWS Graviton (ARM) instance types
+
+In addition to x86 architecture, {hcp-title} offers the following Graviton (ARM) worker node instance types and sizes:
+
+[NOTE]
+====
+Graviton instance types are only available for new clusters created after 22 July, 2024. 
```

Comment:
@arendej is this the date we are using now? I can also put today (19 July) if the feature flags will be enabled.

Reply:
Flag still not enabled. I guess it should be fine :)